tests/test_stdlib.py

```diff
@@ -250,3 +252,33 @@ def test_positional_args_retained(self):
             {'event': '%d %d %s', 'positional_args': positional_args})
         assert 'positional_args' in event_dict
         assert positional_args == event_dict['positional_args']
+
+
+class TestAddLogLevel(object):
+    def test_log_level_added(self):
+        """
+        The log level is added to the event dict.
+        """
+        event_dict = add_log_level(None, 'error', {})
+        assert 'level' in event_dict
```

Comment:
oops sorry i overlooked it in the first round: this is implicitly tested in the next line, let’s keep it simple